internal/supervisor/installation.go

```diff
@@ -1455,3 +1463,23 @@ func getAnnotationsNames(annotations []*model.Annotation) []string {
 func elapsedTimeInSeconds(createAtMillis int64) float64 {
 	return float64(time.Now().Unix()*1000-createAtMillis) / 1000
 }
+
+// migratingHibernateInstallation deeal with dns update for hibernated installations during migration
+func (s *InstallationSupervisor) migratingHibernateInstallation(installation *model.Installation, instanceID string, logger log.FieldLogger) string {
```

Comment:
I think it would be good to adjust the name of this method so that it is straightforward that we are migrating DNS not something else.